ddtrace/span.py

```diff
@@ -169,6 +169,14 @@ def start(self, value):
         # type: (Union[int, float]) -> None
         self.start_ns = int(value * 1e9)
 
+    @property
+    def resource(self):
+        return self._resource[0]
```

Comment:
if `(trace_resource,) = span._resource` is faster, should we do that here to?

```suggestion
        (resource, ) = self._resource
        return resource      
```

This is more a question for @Kyle-Verhoog

Reply:
This comes down to being an implementation detail of the interpreter (I tried in Python 2 and it was about equivalent) so I think we should opt for the more readable, conventional approach by leaving it as-is.

Reply:
👍🏻